### PR TITLE
Fix user menu

### PR DESCRIPTION
**Pull request by pochretien**
move css for the icon provider in the styleguide

### CODE REVIEW

File: resources/icons/svg/domain-google.svg [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+   <path stroke="null" fill="#34A853" id="Shape" d="m23.51621,45.96236c6.32078,0 11.62003,-2.0545 15.49335,-5.5586l-7.56577,-5.75678c-2.09628,1.37661 -4.77784,2.19006 -7.92758,2.19006c-6.09732,0 -11.25823,-4.03599 -13.09912,-9.45901l-7.82118,0l0,5.94449c3.85206,7.49837 11.76902,12.63981 20.92031,12.63981l0,0l0,0.00003z"/>
+   <path stroke="null" fill="#FBBC05" id="Shape" d="m10.41708,27.37805c-0.46821,-1.37662 -0.73423,-2.84709 -0.73423,-4.35929c0,-1.51217 0.26603,-2.98267 0.73423,-4.35928l0,-5.94447l-7.82118,0c-1.5855,3.09738 -2.49,6.6015 -2.49,10.30375c0,3.70226 0.9045,7.20638 2.49,10.30378l7.82118,-5.9445l0,0l0,0.00001z"/>
+   <path stroke="null" fill="#EA4335" id="Shape" d="m23.51621,9.20047c3.43705,0 6.52297,1.1576 8.94911,3.43108l6.71452,-6.58063c-4.05426,-3.70225 -9.35348,-5.97575 -15.66363,-5.97575c-9.15129,0 -17.06825,5.14145 -20.92031,12.63984l7.82118,5.94447c1.84091,-5.42302 7.00182,-9.45901 13.09912,-9.45901l0,0z"/>
+   <path stroke="null" id="Shape" d="m0.1059,0.07517l46.82062,0l0,45.88719l-46.82062,0l0,-45.88719z"/>
+   <g stroke="null" id="handles_square"/>
+  </g>
+ </g>
+</svg>
```
Comment (wfortin) [inline resources/icons/svg/domain-google.svg]: i think this svg can be cleaned up a little. We have many `stroke="null"` that i guess can be removed. `desc` can be removed. I'm not sure but you could try to remove the `defs`

**Comment by wfortin** [comment]
:shipit:

**Comment by wfortin** [comment]
Nice wayyy cleaner :ship:  it

File: docs/_components/components/02_member.html [hunk 2]
```diff
@@ -5,14 +5,14 @@
 stylesheet: '/scss/components/member.scss'
 ---
 <div class="member bg-medium-grey">
-    <img class="member-avatar" src="https://www.gravatar.com/avatar/?d=mm">
-
+    <div>
+        [Provider icon]
```
Comment (marilynechan) [inline docs/_components/components/02_member.html]: The documentation should include the `member-icon` and `member-icon-provider` classes, even if there's no icon displayed.

**Comment by marilynechan** [comment]
Other than my comment for the documentation, nothing to say. Ship it!

**Pull request merged**